primitives/src/identity_role.rs

```diff
@@ -20,7 +20,7 @@ pub enum IdentityRole {
     Investor,
     NodeRunner,
     PM,
-    KYCAMLClaimIssuer,
+    CDDAMLClaimIssuer,
```

Comment:
`CddAmlClaimIssuer` in Pascal case, sorry for being pedantic!

Reply:
No worries, thanks for the advice, this would also be added in my next commit.